### ISSUE TITLE
[Bug]: Exported excel doesn't open while filtering 875+ rows at Highly Confidential Sensitivity

### ISSUE THREAD

**Issue by MrRaghav** [INSIDERS-COOP/pbiviz-docs#14]
### Custom Visual

Mass Filter

### With which Power BI product did it happen?

Power BI Service

### What happened?

When we do not have any sensitivity set, we are able to export as many rows (tested till 20,000!) in the form of excel. However, when we have sensitivity set to Highly Confidential, export happens but excel does not open if the number for rows are more than 875. 

### Contact Details

raghavs1003@gmail.com

**Comment by j-lebrun** [comment]
Hi @MrRaghav,

Thank you for this feedback.

Is it specificly related to the Mass Filter?
Do you have the same issue with other slicers?

Is 875 a particular value?
The number of remaining non sensitive rows after filtering, maybe?

**Comment by MrRaghav** [comment]
Yes, it is specifically with Mass filter as we have used it to filter more IDs. No such issue with any other default slicer in Power BI. 

Yes, any number of IDs more than 860/870 have this issue.

No, all of them are sensitive.
